docs/native/src/scenarios/from_app.rst

```diff
@@ -9,42 +9,45 @@ Opening files from your app in |Office iOS|
 In order to invoke |Office iOS| when opening an Office file from your app, you must use the URL schemes that Word,
 Excel, and PowerPoint for iOS register when they are installed.
 
-General information on the Office URL schemes can be found here:
-https://msdn.microsoft.com/en-us/library/office/dn906146.aspx#sectionSection1
+URL Schemes for Inovking Office
```

Comment:
I believe I fixed all the issues you provided feedback on.
But I’m not sure how to do the check_spelling.bat though. I have the GitHub app which really only helps me manage the branches, not with actual editing. check_spelling doesn’t seem to work in powershell.

Can you provide some pointers on how to do this?

Thanks,
Karvell

From: Tyler Butler [mailto:notifications@github.com]
Sent: Thursday, February 11, 2016 6:24 PM
To: Microsoft/Office-Online-Test-Tools-and-Documentation <Office-Online-Test-Tools-and-Documentation@noreply.github.com>
Cc: Karvell Li <karvell@microsoft.com>
Subject: Re: [Office-Online-Test-Tools-and-Documentation] Update from_app.rst (#173)


In docs/native/src/scenarios/from_app.rst<https://github.com/Microsoft/Office-Online-Test-Tools-and-Documentation/pull/173#discussion_r52699441>:

> @@ -9,42 +9,45 @@ Opening files from your app in |Office iOS|

>  In order to invoke |Office iOS| when opening an Office file from your app, you must use the URL schemes that Word,

>  Excel, and PowerPoint for iOS register when they are installed.

>

> -General information on the Office URL schemes can be found here:

> -https://msdn.microsoft.com/en-us/library/office/dn906146.aspx#sectionSection1

> +URL Schemes for Inovking Office

Typo. If you set up a local build environment you can actually run check_spelling.bat to check your spelling. [:smile:]

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/Office-Online-Test-Tools-and-Documentation/pull/173/files#r52699441>.